src/Task/Base/ParallelExec.php

```diff
@@ -172,6 +188,12 @@ public function run()
                         }
                     }
                     unset($running[$k]);
+                    if ($this->stopOnFail && $process->getExitCode() !== 0) {
+                        foreach ($running as $otherRunningProcess) {
+                            $otherRunningProcess->stop();
```

Comment:
This can't be done without prefixing everything with `exec` command, read:

1. https://symfony-documentations.readthedocs.io/en/2.6/components/process.html#process-signals
1. https://github.com/symfony/symfony/issues/5759
1. https://bugs.php.net/bug.php?id=39992

Reply:
Hm, I am ambivalent about this. Would it be possible to use `posix_setpgid` as described in the PHP bug report to fix things for those who have the posix commands?

Reply:
I'll try